Exercise_A/index.html

```diff
@@ -6,12 +6,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title></title>
     <meta name="description" content="" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+
   </head>
   <body>
     <!-- CODE HERE -->
+    <nav class="navbar navbar-expand-lg navbar-light">
+      <img src="/Exercise_A/assets/bootstrap.png" width="50" height="50" alt="Button">

```

Comment:
The image didn't load on the page due to the directory not being exactly specific. As the image was inside a folder within Exercise A, we have to put ../Exercise_A/assets/bootstrap.png then it will grab the image and load it on our page.